src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,14 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+    public Colori getRandomColor() {
+
+        int randomIndex = new Random().nextInt(Colori.values().length);
+        return Colori.values()[randomIndex];
```

Comment:
Nice use of the `values()` method of the enum to get all its values. Remember that this method creates a new array each time it's called. To avoid this overhead, you can store the values in a class field and reuse it. This is a performance issue.